log/sequencer.go

```diff
@@ -22,6 +23,11 @@ type Sequencer struct {
 	timeSource util.TimeSource
 	logStorage storage.LogStorage
 	keyManager crypto.KeyManager
+
+	// These parameters can be safely adjusted during operation
```

Comment:
OK; we run Travis tests with -race so if this does start to get used in a racy way we should get told about it.